server/controllers/request.js

```diff
@@ -0,0 +1,97 @@
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+const Request = require("../models/Request");
+
+// @route GET /request
+// @desc gets all requests for loged-in dog sitter
+// @access Private
+exports.getRequests = asyncHandler(async (req, res, next) => {
+    const { id: sitterId } = req.user; 
+    
+    const requests = await Request.find({ sitterId })
+    .populate('userId')
+    .populate("sitterId")
```

Comment:
If we are populating the user ids here, won't that send the password from the user document in the response as well? Although the password is hashed, this is not advisable from a security standpoint :)

Reply:
Thanks. Fixing that in the next commit